README.md

```diff
@@ -1,5 +1,170 @@
+  </tr> 
+</table>
+
+<br/>
+
+## Features 🌟
+
+- **Sync with Google Drive**:  Easily access and manage your photos and videos stored on your device
+  and Google Drive account directly in one place. Enjoy the convenience of having your memories at
+  your fingertips, wherever you go.
+- **Auto Backup In Google Drive**: Worried about losing your cherished memories? Cloud Gallery's
+  auto backup feature ensures that your local photos and videos are automatically backed up to your
+  Google Drive account, keeping them safe and secure.
+- **Effortless Uploads & Downloads**: Effortless Uploads & Downloads: Easily upload photos and
+  videos from your device's local storage to your Google Drive account, and download files from
+  Google Drive to your device with just a tap.
+- **Access media in Multiple Devices**: Access your media seamlessly across multiple devices with
+  our backup feature, ensuring your content is always at your fingertips
+- **Stay Organized**: Arrange and categorize your photos and videos in your Google Drive account,
+  making it easy to find and manage your media files.
+- **Secure User Data**: We prioritize your privacy and security. Cloud Gallery utilizes your Google
+  Drive storage, ensuring that your data remains secure within your own account.
+- **Cross-Platform Compatibility**: Whether you're using an Android or iOS device, Cloud Gallery
+  offers seamless compatibility across platforms.
+- **User-Friendly Interface**: With an intuitive and user-friendly interface, Cloud Gallery makes it
+  easy to manage your media files, ensuring a hassle-free experience.
+
+> **Note**: Cloud Gallery is currently in active development 🚧, with plans to incorporate additional
+> features shortly. Stay tuned for updates!
+
+<br/>
+
+## What's Coming Next? 🚀
+
+- **Dropbox Integration**: Cloud Gallery will soon support Dropbox integration, allowing users to
+  access and manage their media files dropbox.
+- **Sharing Between Clouds**: Easily share your media files between Google Drive and Dropbox, making
+  it convenient to access and manage your files across multiple cloud providers.
+- **Auto Backup In Multiple Cloud**: Cloud Gallery will soon support backup in multiple cloud
+  providers, ensuring that your media files are securely stored across multiple platforms.
+- **Enhanced Security Features**: We're working on enhancing our security features to ensure that
+  your data remains safe and secure.
+- **Improved User Experience**: We're continuously working on improving the user experience, with
+  plans to incorporate additional features and enhancements shortly.
+- **Bug Fixes & Performance Improvements**: We're actively working on addressing any bugs and
+  performance issues to ensure a seamless user experience.
+- **Feedback & Suggestions**: We value your feedback and suggestions. If you have any ideas or
+  suggestions, please let us know. We'd love to hear from you!
```

Comment:
I think this part should be removed from here and can be added somewhere else.
We can add support section for this